h/static/scripts/group-forms/components/GroupFormHeader.tsx

```diff
@@ -0,0 +1,48 @@
+import classnames from 'classnames';
+import type { ComponentChildren } from 'preact';
+import { Link as RouterLink, useRoute } from 'wouter-preact';
+
+import type { Group } from '../config';
+import { routes } from '../routes';
+
+type TabLinkProps = {
+  href: string;
+  children: ComponentChildren;
+};
+
+function TabLink({ children, href }: TabLinkProps) {
+  const [selected] = useRoute(href);
+  return (
+    <RouterLink
+      href={href}
+      className={classnames({
+        'focus-visible-ring whitespace-nowrap flex items-center font-semibold rounded px-2 py-1 gap-x-2':
+          true,
+        'text-grey-1 bg-grey-7': selected,
+      })}
```

Comment:
Maybe you want to look into https://patterns.hypothes.is/navigation-tab, although, I don't know how much it allows the UI to be customized, but it adds keyboard navigation and such.